src/main/java/uk/ac/ebi/embl/api/validation/check/file/FileValidationCheck.java

```diff
@@ -687,4 +688,15 @@ public static boolean isHasAgp() {
 	public static void setHasAgp(boolean hasAgpFiles) {
 		hasAgp = hasAgpFiles;
 	}
+
+	boolean validateSequenceCountForTemplate(ValidationResult validationResult, SubmissionFile submissionFile) {
```

Comment:
- I would consider moving validateSequenceCountForTemplate to TSVFileValidationCheck if we only use it there.